## This is a projetc about exceptions (try, except) in Python.

| # | Function | Description |
| - | ------ | ------ |
| 0 | safe_print_list | This function prints **x** elements of a list (using try: / except:). |
| 1 | safe_print_integer | This function Write a function that prints an integer with "{:d}".format(). |
| 2 | safe_print_list_integers | This function prints the first **x** elements of a list and only integers. |
| 3 | safe_print_division | This function divides 2 integers and prints the result (using try: / except: / finally:). |
| 4 | list_division | This function  divides element by element 2 lists. |
| 5 | raise_exception | This function raises a type exception. |
| 6 | raise_exception_msg | This function raises a name exception with a message. |
| 100 | safe_print_integer_err | This function prints an integer. If exception prit in stderr. |
| 101 | safe_function | This function executes a function safely. |
| 102 | magic_calculation | This function does exactly the same as the following Python bytecode. |
